frontend/view-edit-client/client-history/client-history.component.tsx

```diff
@@ -207,28 +207,25 @@ export default function ClientHistory(props: ClientHistoryProps) {
   }
 
   function getDate(log, index) {
+    const previousTimestampDate =
```

Comment:
This was a bug caused by our work a couple weeks ago in #693  - this function threw an error when index was 0 because it looks up `index - 1`. The reason it normally doesn't happen is that the first entry is usually "client added to database." However, that doesn't always hold true, since you can change dateOfContact in follow ups.